internal/action/sync.go

```diff
@@ -128,6 +136,14 @@ func (s *Action) sync(ctx context.Context, store string) error {
 
 // syncMount syncs a single mount.
 func (s *Action) syncMount(ctx context.Context, mp string) error {
+	// using GetM here to get the value for this mount, it might be different
+	// than the global value
+	if as := s.cfg.GetM(mp, "core.autosync"); as == "false" {
```

Comment:
Note that the current `GetBool` implementation assumes that `"true"` will mean true and any other value will mean `false` whereas here is assumes `"false"`. 
A slight difference that could bite since `GetBool` hasn't the same behaviour.

Reply:
This is true. A bit tricky. But I still think the current implementation is OK.

I'd assume the default to be "autosync all stores" or maybe "autosync no stores".
But we should be able to support both "autosync no stores except this one" and "autosync all stores except this one".

Reply:
An easy fix to have both match the GetBool behaviour would be:
```suggestion
	if as := s.cfg.GetM(mp, "core.autosync"); as != "true" {
```